packages/happn-3/test/__fixtures/test/temp_test_lookup_db

```diff
@@ -0,0 +1 @@
+{"snapshot":"{\"filename\":\"loki.db\",\"collections\":[{\"name\":\"happn\",\"data\":[],\"idIndex\":null,\"binaryIndices\":{\"path\":{\"name\":\"path\",\"dirty\":false,\"values\":[]},\"created\":{\"name\":\"created\",\"dirty\":false,\"values\":[]},\"modified\":{\"name\":\"modified\",\"dirty\":false,\"values\":[]}},\"constraints\":null,\"uniqueNames\":[\"path\"],\"transforms\":{},\"objType\":\"happn\",\"dirty\":true,\"cachedIndex\":null,\"cachedBinaryIndex\":null,\"cachedData\":null,\"adaptiveBinaryIndices\":true,\"transactional\":false,\"cloneObjects\":false,\"cloneMethod\":\"parse-stringify\",\"asyncListeners\":false,\"disableMeta\":false,\"disableChangesApi\":true,\"disableDeltaChangesApi\":true,\"autoupdate\":false,\"serializableIndices\":true,\"disableFreeze\":true,\"ttl\":null,\"maxId\":0,\"DynamicViews\":[],\"events\":{\"insert\":[],\"update\":[],\"pre-insert\":[],\"pre-update\":[],\"close\":[],\"flushbuffer\":[],\"error\":[],\"delete\":[null],\"warning\":[null]},\"changes\":[],\"dirtyIds\":[]}],\"databaseVersion\":1.5,\"engineVersion\":1.5,\"autosave\":false,\"autosaveInterval\":5000,\"autosaveHandle\":null,\"throttledSaves\":true,\"options\":{\"serializationMethod\":\"normal\",\"destructureDelimiter\":\"$<\\n\"},\"persistenceMethod\":\"fs\",\"persistenceAdapter\":null,\"verbose\":false,\"events\":{\"init\":[null],\"loaded\":[],\"flushChanges\":[],\"close\":[],\"changes\":[],\"warning\":[]},\"ENV\":\"NODEJS\"}"}

```

Comment:
We need to investigate why this is still landing up there - do a grep for test_lookup_db, maybe the test unlinks it, but not the temp file